update/util.go

```diff
@@ -83,3 +85,24 @@ func RemoveNilErrors(errs []error) []error {
 	}
 	return r
 }
+
+// RandString returns random (base32) string with prefix
+func RandString(numBytes int) (string, error) {
+	buf, err := RandBytes(numBytes)
+	if err != nil {
+		return "", err
+	}
+	str := base32.StdEncoding.EncodeToString(buf)
```

Comment:
I'm not sure if `EncodeToString` does exactly what we need here. The result of encoding a single zero byte is `AE======`. We probably don't want the padding, and also the final character isn't fully random. I don't think this is a problem if we stick to multiples of 5 in the length, and it looks like you're using 20 above, but I'd worry about changing that length in the future.